source/ui/BioXAS/BioXASSSRLMonochromatorRegionControlEditor.cpp

```diff
@@ -2,7 +2,7 @@
 #include "beamline/BioXAS/BioXASSSRLMonochromatorRegionControl.h"
 
 BioXASSSRLMonochromatorRegionControlEditor::BioXASSSRLMonochromatorRegionControlEditor(BioXASSSRLMonochromatorRegionControl *regionControl, QWidget *parent) :
-	BioXASControlEditor(regionControl, parent)
+	CLSControlEditor(regionControl, parent)
 {
 
 }
```

Comment:
If it were up to me, I would prefer to handle changes to BioXASControlEditor as a separate issue. I'm not sure it's ready for use throughout the CLS yet--I'm still doing some development work on it.